src/chapters/aspdotnet-model-validation/viewmodels.rst

```diff
@@ -0,0 +1,91 @@
+.. index:: ! ViewModel
+
+ViewModels and Passing Data To and From Views
+=============================================
+
+Now that we have an understanding for what a model is, we can focus on how to effectively pass information between the three elements of MVC applications.
+With our current MVC application, we can add new events and remove events.
+However, our application is also susceptible to run-time errors.
+Our view can accept any type of input and if we mistype something in our view, we can run into issues later down the line.
+
+A **ViewModel** is a model class designed specifically to be used in a view.
+By utilizing ViewModels in our application, we can make our views strongly-typed and add validation to forms to prevent bad user input.
+These benefits of ViewModels will help reduce potential errors in our application.
+
+Refactoring the View and Controller to Use a Model in a View - Video
+--------------------------------------------------------------------
+
+.. TODO: Add video covering an intro about how to use a model in a view
+
+.. admonition:: Note
+
+   If you want to verify what code this video starts with, check out the `model-binding <https://github.com/LaunchCodeEducation/CodingEventsDemo/tree/model-binding>`__ branch.
+   If you want to verify what code this video ends with, check out the `models-in-views <https://github.com/LaunchCodeEducation/CodingEventsDemo/tree/models-in-views>`__ branch.
+
+Refactoring the View and Controller to Use a Model in a View - Text
+-------------------------------------------------------------------
+
+To start with understanding why we may want to use a ViewModel, let's refactor our code to use a model directly in our view.
+
+First, we want to convert the collection of Events we have been holding into a list.
+
+.. sourcecode:: csharp
```

Comment:
Which file is this in? Will it be obvious to the reader? If not, it might be worth mentioning explicitly.

Reply:
Done!